run

```diff
@@ -3,7 +3,7 @@
 JAR=build/libs/app.jar
 HEAP_SIZE=2g
 
-[ -z ${JVM_OPTIONS} ] && JVM_OPTIONS="-server -Xms${HEAP_SIZE} -Xmx${HEAP_SIZE} --add-opens=java.base/java.time=ALL-UNNAMED --add-modules=jdk.incubator.vector"
+[ -z ${JVM_OPTIONS} ] && JVM_OPTIONS="-server -Xms${HEAP_SIZE} -Xmx${HEAP_SIZE} --add-opens=java.base/java.time=ALL-UNNAMED --add-modules=jdk.incubator.vector -Dfastjson2.features=disableReferenceDetect,disableArrayMapping,disableJSONB,disableAutoType,disableSmartMatch"
```

Comment:
Are these fastjson2 options documented somewhere? and why they are safe to use in a production environment? Why would they not be set by default if they are safe?

Moreover, it would be preferable to introduce Java code/class (via System.setProperty) to increase the visibility that these options are used, and optionally to find a way to run fastjson2_optimized vs fastjson2_default.

Reply:
fastjson provides many functions that other json libraries do not have. These options can be turned off on demand and can be used in a production environment. There is a [Chinese document](https://github.com/alibaba/fastjson2/wiki/jit_optimization) introducing it.

The current version only supports global opening or closing. Serde that provides per-instance configuration options will be provided in the next version (2.0.51)

Reply:
Let's wait for 2.0.51 then. When it's ready, let's introduce a new FASTJSON2_OPTIMIZED implementation in Library, etc. so the two can be compared side by side. I read the docs and using these flags seem to require in-depth knowledge of certain fastjson2 features to be useful, so best to keep it isolated from the default fastjson2 behavior (whatever that may be).